content/chainguard/chainguard-images/security-advisories/index.md

```diff
@@ -0,0 +1,161 @@
+
+
+## Searching the Security Advisories
+
+After finding a vulnerability in a Chainguard Image, you can navigate to [Chainguard's **Security Advisories** page](https://images.chainguard.dev/security). This is a helpful resource you can use to determine the status for any CVE found within a Chainguard Image.
+
+The Security Advisories page is self-service, allowing you to check whether Chainguard is aware of a specific vulnerability and whether it has been mitigated in a certain package version. You can search the Security Advisories page by entering any CVE identifier to find what packages are affected by that CVE. You can also enter the names of individual packages to find what CVEs have been reported within them.
+
+Enter the CVE identifier you copied previously into the search box at the top of the page. This will immediately filter the list of security advisories to only show packages where that CVE has been reported. It will also show the **Status** of each.
+
+![Screenshot showing the Chainguard Security Advisories page. The list of advisories has been filtered to only include those with a CVE ID of "CVE-2023-44487".](sec-advisories-1.png)
+
+If you click on any row in the filtered list, it will take you to the CVE's specific page. There, you'll find a list of every package where this CVE has been reported.
+
+![Screenshot showing the Security Advisory page for CVE-2023-44487 and a partial list of its affected packages.](sec-advisories-2.png)
+
+As with the Security Advisories landing page, you can filter by package name here on the CVE's landing page as well. Enter the name of the package we highlighted previously (`nghttp2`) and the index will immediately filter out any packages that do not mention that string in their metadata.
+
+![Screenshot showing a filtered list of packages affected by CVE-2023-44487, this time only showing packages that reference "nghttp2". The row for the nghttp2 package itself shows a "Status" of "Fixed", and a "Fixed Version" of "1.57.0-r0".](sec-advisories-3.png)
+
+As the previous screenshot highlights, for CVE-2023-44487, the `nghttp2` package's **Status** is marked as **Fixed** in version `1.57.0-r0` as of October 11, 2023.
+
+
+## Comparing Images
+
+Chainguard's Security Advisories have told us that the CVE-2023-44487 was fixed and removed from `nghttp2` with a more recent version than the one available in Chainguard's `go:1.21.2` Image. However, we don't have to take that report at face value; we can inspect a later version of the same Image and compare it with version `1.21.2` to determine whether the vulnerability is still present in the later version.
+
+If you inspect a later version of the Image with Docker Scout, you'll find that this time it reports no CVEs. This example inspects version `1.21.5` of the Image.
```

Comment:
Probably worth mentioning that at the time of writing 1.21.5 was current, but it might not be now, and may have since accumulated CVEs.

Reply:
great call